securebanking-openbanking-uk-rcs-sample/docker/docker-compose.yml

```diff
@@ -0,0 +1,82 @@
+#
```

Comment:
The default docker-compose file, which retrieves config using the native file system based profile.